pkg/engine/api/builders/kcl/kcl_builder.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path"
```

Comment:
Why is `path`, not `path/filepath`

Reply:
This is moved directly from the `cmd` pkg